zstacklib/zstacklib/utils/http.py

```
@@ -303,7 +303,16 @@ def _build(self):
 
         cherrypy.engine.autoreload.unsubscribe()
         site_config = {}
-        site_config['server.socket_host'] = '0.0.0.0'
+        # IPv6 双栈监听：'::' 在 Linux(IPV6_V6ONLY=0)下同时接受 IPv4/IPv6 连接
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)